app/components/comment-item.js

```diff
@@ -72,15 +75,16 @@ export default Component.extend({
   /**
    * nullbody - Computed property
    * 
-   * checks to see if outcome of processed content is null, indicating null comment body
-   * and returns computed property as true if body is null.
+   * checks to see if outcome of processed content is null, indicating null 
+   * comment body and returns computed property as true if body is null.
    * 
-   * NOTE: only did it this way because there is a test that lead me to think that this
-   * component could possibly load before comment body is loaded. This will prevent the 
-   * "No description provided" message from showing up on load before data is available.
+   * It tests specifically against null because parsedBody generates other 
+   * falsey outputs even when comment body is not null. 
+   * For example is if comment data is not loaded yet, 
+   * parsedBody trurns undefined
```

Comment:
Would reword as:

> It tests specifically against `null` because `parsedBody` generates other falsey outputs even when comment body is not `null`. For example, if the comment data is not loaded yet, `parsedBody` returns `undefined`.